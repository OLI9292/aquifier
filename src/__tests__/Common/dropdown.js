import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import sinon from 'sinon';

import Dropdown from '../../Components/Common/dropdown';

const choices = ['Earth', 'TAG'];

describe('Dropdown', () => {
  it('should render correctly', () => {
    const output = shallow(
      <Dropdown
        choices={choices}
        selected={choices[0]} />
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
})
