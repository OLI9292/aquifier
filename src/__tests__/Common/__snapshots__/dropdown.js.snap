// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown should render correctly 1`] = `
<styled.div>
  <styled.p
    key="Earth"
    onClick={[Function]}
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
    selected={true}
    show={true}
  >
    Earth
    <img
      alt="down arrow"
      src="white-arrow-down.png"
      style={
        Object {
          "height": "45%",
          "margin": "12px 0px 0px 5px",
        }
      }
    />
  </styled.p>
  <styled.p
    key="TAG"
    onClick={[Function]}
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
    selected={false}
    show={false}
  >
    TAG
    <img
      alt="down arrow"
      src="white-arrow-down.png"
      style={
        Object {
          "height": "45%",
          "margin": "12px 0px 0px 5px",
        }
      }
    />
  </styled.p>
</styled.div>
`;
